homeassistant/components/alert/__init__.py

```diff
@@ -78,12 +80,13 @@ def is_on(hass, entity_id):
         can_ack = cfg.get(CONF_CAN_ACK)
         title_template = cfg.get(CONF_TITLE)
         data = cfg.get(CONF_DATA)
+        notifier_data = cfg.get(CONF_NOTIFIER_DATA)
```

Comment:
Please use `dict[key]` for required config keys and keys with default config schema values.